app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt

```diff
@@ -20,11 +20,39 @@ package net.skyscanner.backpack.demo.stories
 
 import android.os.Bundle
 import android.view.View
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
 import net.skyscanner.backpack.barchart.BpkBarChart
 import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.components.BarChartComponent
+import net.skyscanner.backpack.demo.meta.ViewStory
+import net.skyscanner.backpack.demo.ui.AndroidLayout
 import java.util.Random
 
-class BarChartStory : Story() {
+@Composable
+@BarChartComponent
+@ViewStory
+fun BarChartStory(modifier: Modifier = Modifier) =
+  AndroidLayout<BpkBarChart>(R.layout.fragment_bar_chart, R.id.bar_chart, modifier.fillMaxSize()) {
```

Comment:
Some of the view stories require fillMaxSize(), otherwise, it won't be taking the full screen even despite the layout params being set to match_parent. The main reason why is whenever you use LayoutInflater, it uses parent to correctly calculate match_parent values – and we don't have parents in AndroidView